docker/Dockerfile.local

```diff
@@ -9,7 +9,7 @@ RUN apt-get install -y python3-pip git wget psmisc
 RUN apt-get install -y cmake
 
 # Install Pytorch
-RUN pip install torch==1.13.1+cu116 --extra-index-url https://download.pytorch.org/whl/cu116
+RUN pip3 install torch==2.1.0 torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
```

Comment:
Do we really need torchvision or torchaudio for any use? Maybe we need a double check then.

Reply:
Yeah..lets get rid of them.